src/site/content/en/blog/samesite-cookie-recipes/index.md

```diff
@@ -0,0 +1,324 @@
+
+While "unsafe" may sound slighly concerning here, this refers to any request
+that may be intended to change state and on the web that's primarily POST
+requests. Cookies marked as `SameSite=Lax` will be sent on safe top-level
+navigations, e.g. clicking a link to go to a different site. However something
+like a `<form>` submission via POST to a different site would not include
+cookies.
+
+<figure class="w-figure  w-figure--center">
+  <img src="safe-navigation.png"
+      alt="Diagram of a request moving from one page to another."
+      style="max-width: 35vw;">
+  <figcaption class="w-figcaption">
+    If the incoming request uses a "safe" method then the cookies will be sent.
+  </figcaption>
+</figure>
+
+This pattern is used for sites that may redirect the user out to a remote
+service to perform some operation before returning, for example redirecting to a
+third-party identity provider. Before the user leaves the site, a cookie is set
+containing a single use token with the expectation that this token can be
+checked on the returning request to mitigate CSRF attacks. If that returning
+request comes via POST then it will be necessary to mark the cookies as
+`SameSite=None; Secure`.
+
+### Remote resources
+
+Any remote resource on a page may be relying on cookies to be sent with that
+request, from `<img>` tags, `<script>` tags, and so on. Common use cases here
+include tracking pixels and personalizing content.
+
+This also applies to
+[`fetch` requests](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch#Sending_a_request_with_credentials_included)
+from the page. If you see the `credentials: 'include'` option in the
+configuration this is a good indication that cookies may well be expected on
+those requests. Those cookies will need to be appropriate marked to be included.
+
+## How to implement `SameSite` today
+
+For cookies where you know they are only needed in a first-party context you
+should ideally mark them as `SameSite=Lax` or `SameSite=Strict` depending on
+your needs. You can also choose to do nothing and just allow the browser to
+enforce its default, but this comes with the risk of inconsistent behavior
+across browsers and potential console warnings for each cookie.
+
+```text
+Set-Cookie: first_party_var=value; SameSite=Lax
+```
+
+For cookies needed in a third-party context, you will need to ensure thet are
+marked as `SameSite=None; Secure`. Note that you need both attributes together.
+If you just specify `None` without `Secure` the cookie will be rejected. There
+are some mutually incompatible differences in browser implementations here
+though, so you may need to use some of the mitigating strategies described in
+["Handling incompatible clients"](#handling-incompatible-clients) below.
+
+```text
+Set-Cookie: third_party_var=value; SameSite=None; Secure
+```
+
+### Identifying cookie usage
+
+As of Chrome 77, you will be able to see warnings in the console cross-site
```

Comment:
Delete 'will be able to'.

Missing a word at the end: 'in the console **for** cross-site'.

Reply:
Fixed.